src/styles/blocks/icon.scss

```diff
@@ -0,0 +1,31 @@
+.icon {
+  display: block;
+  background-size: cover;
+  height: 24px;
+  width: 24px;
+
+  @include hover(transform, scale(1.2));
+
+  @include on-tablet {
+    height: 32px;
+    width: 32px;
+  }
+
+  &--phone {
+    background-image: url(/src/images/icons/Icon-Phone-call.svg);
+  }
+
+  &--menu {
+    background-image: url(/src/images/icons/Icon-Burger-menu.svg);
```

Comment:
[Not critical]
The approach with having separate images for regular and hover states works fine. But since the difference in their sizes is not big, I'd better consider using the same image, but with different width/height for these states.